README.md

```diff
@@ -29,3 +29,4 @@
 - Hello I am Shourjjo Majumder, An indie software and game developer, I mainly work with Python and C and currently also intrested in electronics. - [shour](https://github.com/shourdev)
 - Hello! I am Adwait Kulkarni, a 3rd-year computer science major studying at the University of British Columbia in Vancouver, Canada. I am proficient in full-stack development and have experience with the MERN stack. Currently I am working with Java backend, particularly Spring Boot and building apps using the same. [Adwait Kulkarni](https://github.com/AdwaitKulkarni58)
 - Good Evening my name is Anubhav and I am an aspiring developer and a problem solver with expertise in various technologies. I have contributed to numerous projects and have gained recognition for my work. I consider myself fairly creative and a hardworking individual. I have a passion for technology and gaming. I have hands-on experience in building real-world projects and keep myself updated with all the news related to gaming and tech. When I’m not coding, you can find me playing games and exploring new technologies. - [Anubhav Adhikari](https://github.com/anubhav1206)
+- Hey Everyone, myself Sivasai and I am web developer with lot of enthusiasm to learn new things on the go. I know C++, Javascript, React, GoLang, MongoDB and Solidity. I am a noob in opensource. Check out my profile -[sivasaisilla](https://github.com/sivasaisilla)
```

Comment:
I don't really like that you state that your a noob in opensource, that's not the right words and it makes you sound bad. Instead, let's put New To Open-Source :)!, what do you say?

Reply:
Sure @KendallDoesCoding